packages/projects-docs/pages/learn/sandboxes/editors.mdx

```diff
@@ -10,139 +10,19 @@ import Video from '../../../../../shared-components/Video'
-    
-
-
-## Uploading Static Files
-
-It's sometimes desired to have either images or big files in a sandbox. We allow
-you to upload these within CodeSandbox. As a
-[patron](https://codesandbox.io/patron) you are allowed to upload a maximum of
-500MB, otherwise you are able to upload 20MB. Remember that all uploaded files
-are public, regardless of the privacy of the sandbox
-
-<br/>
-### Ways to upload
-There are two ways to upload files: by using a wizard or by drag and dropping
-the files in a folder in CodeSandbox. The wizard can be opened by clicking on
-the upload icon in the Files tab.
-
-![Upload Wizard](../images/upload.jpg)
-
-Dragging and dropping the files in the Files tab works as well.
-
-<br/>
-### Storage Management
-
-You can use an uploaded file in multiple sandboxes at the same time. For this
-reason we have a separate 'Storage Management' that allows you to see which
-files you have uploaded. You can find the manager in your user menu.
-
-![Storage Management in User Menu](../images/storage-management-users.jpg?v2)
-
-You are able to view your storage usage with the option to either open uploaded
-files, add uploaded files to the opened sandbox or to delete an uploaded file.
-
-![Storage Management](../images/storage-management.jpg)
-
-<br/>
-### CLI Integration
-
-When you deploy a sandbox using the CLI we automatically detect the binary files
-and upload them to the CodeSandbox Storage Management after getting
-confirmation.
-    </WrapContent>
-    <WrapContent>
-    
+<Tabs tabs={["Cloud", "Browser"]}>
+    <WrapContent>    
 Cloud sandboxes run inside a Firecracker MicroVM and utilise memory snapshotting to ensure fast fork time and wake up time. They can run anything that you would be able to run in your local environment, including Docker containers. You can learn more about how they work [here](/blog/how-we-clone-a-running-vm-in-2-seconds).
 
 Cloud sandboxes are best used to run backend services, compiled languages and things like databases.
 
-The Cloud Sandbox Editor contains a set of features to increase the accessibility of complex tools like git, setup, review flows, contributions, and visualizing components. 
-
-Here is an overview of the editor structure:
-
-### Header
-
-![The web editor](../images/overview-header.jpg)
-
-The header is where you perform general operations on your Sandbox.
-
-**1. Navigation**
-    - **Sandbox**: Click to return to the dashboard.
-
-**2. Devtool menu**
-    - **Add Devtools**: Set up devtools to facilitate your work and automatize repetitive tasks.
-
-**3. Connections**
-    - **Avatars**: See who's currently online and click to follow their steps.
-
-**4. Collaborate**
-    - **Share**: Invite friends to work together.
+The Cloud Sandbox editor contains a set of features to increase the accessibility of complex tools like git, setup, review flows, contributions, and visualizing components. 
 
+    </WrapContent>
```

Comment:
Since we split browser sandboxes into a new section, we should probably remove this from here (and consider moving it to a new "Editor" subsection inside Browser Sandboxes -> Features.)

Reply:
good call